install/install.go

```diff
@@ -236,7 +241,12 @@ func K8sInstaller(c *k8s.Client, o Options) error {
 			},
 		}
 		if _, err := c.K8sClientset.CoreV1().Namespaces().Create(context.Background(), &newns, metav1.CreateOptions{}); err != nil {
-			return fmt.Errorf("failed to create namespace %s: %+v", ns, err)
+			if o.Save {
+				fmt.Printf("⚠️\tfailed to create namespace %s: %+v\n", ns, err)
```

Comment:
I think if `save` is enabled then karmor should not be creating any k8s resources including namespace ... 

The logic should be to bypass namespace create if `save` is attempted.

@daemon1024 WDYT?

Reply:
That's correct.

Reply:
Ok, will do the required changes